README.md

```diff
@@ -27,23 +27,23 @@ https://github.com/hudmol/user_defined_in_basic/releases
 Add an entry to your `config.rb` like this:
```

Comment:
Changes in here derive from me changing the way it works out what record type we're looking at.  Since I had to move the logic out into the controller, I thought I might as well just use the controller name.  That means a slight change to the config file and the documentation, but has the nice bonus of allowing the plugin to support any future record types that come along...